ecmean.py

```diff
@@ -299,18 +299,28 @@ def getcomponent(face):  # unused function
 def getinifiles(face, diag):
     """Return the inifiles from the interface, needs the component dictionary"""
     comp = face['model']['component']
-    atminifile = os.path.expandvars(face['component'][comp['atm']]
-                                        ['inifile'].format(expname=diag.expname))
-    oceinifile = os.path.expandvars(face['component'][comp['oce']]
-                                        ['inifile'].format(expname=diag.expname))
-    if not atminifile[0] == '/':
-        atminifile = str(diag.ECEDIR /
-                         Path(os.path.expandvars(face['model']['basedir'].format(expname=diag.expname))) /
-                         Path(atminifile))
-    if not oceinifile[0] == '/':
-        oceinifile = str(diag.ECEDIR /
+
+    # use a dictionary to create the list of initial files
+    inifiles = {}
+    for k, v in zip(['atminifile','ocegridfile','oceareafile'],['inifile','gridfile','areafile']) :
+
+        # the component is atmosphere for atminifile only
+        x = 'atm' if k in 'atminifile' else 'oce'
```

Comment:
Two things: 
- instead of "x" it would be more pythonic to use something more expressive like "component"
- I would prefer to write this as
```
if k == 'atminifile':
    component = 'atm'
else:
   component = 'oce'

```

Reply:
using "`if k in something`"  where "something" is only one thing, to say  "`if k == something`" looks like an abuse to me.

Reply:
Agreed!

Reply:
done as well